services/packagist/packagist-php-version.tester.js

```diff
@@ -11,6 +11,14 @@ t.create('gets the package version of symfony 2.8')
   .get('/symfony/symfony/v2.8.0.json')
   .expectBadge({ label: 'php', message: isComposerVersion })
 
+t.create('package with no requirements')
+  .get('/bpampuch/pdfmake.json')
+  .expectBadge({ label: 'php', message: 'version requirement not found' })
+
+t.create('package with no php version requirement')
+  .get('/raulfraile/ladybug-theme-modern.json')
```

Comment:
👍 How'd you go about finding these packages to use for the tests (in case we have to find new example packages down the road0?

Reply:
Actually, this wasn't too hard: the programming language is indicated when you search for packages through the UI:
![packagist](https://user-images.githubusercontent.com/10694593/61828962-cca4f680-ae5f-11e9-8397-e7e6ab5f7744.png)
Most are PHP, but some aren't and are therefore unlikely to require any specific PHP version, and quite a few don't have any dependencies at all. So I searched for some random keywords and kept on scrolling until another programming language appeared. 😄